src/test/fuzz/fuzz.cpp

```diff
@@ -25,6 +25,7 @@ static bool read_stdin(std::vector<uint8_t>& data)
 static void initialize()
 {
     const static auto verify_handle = MakeUnique<ECCVerifyHandle>();
+    SelectParams(CBaseChainParams::REGTEST);
```

Comment:
https://github.com/bitcoin/bitcoin/pull/17071#discussion_r332063071

Reply:
Looks like chain selection might not be needed for these fuzzers: removed. Please re-review.